libs/redux/src/lib/stores/dashboard/dashboard-store-helpers.ts

```diff
@@ -11,7 +11,7 @@ export const filterPromptsBySearch = (
       if (searchKey && searchValue) {
         return (
           key.toLowerCase() === searchKey.toLowerCase() &&
-          value.toLowerCase() === searchValue.toLowerCase()
+          value.toLowerCase().includes(searchValue.toLowerCase())
```

Comment:
Not sure if you're trimming higher up in the call stack, but can help the input avoid some various edge cases for the match.
```suggestion
          value.toLowerCase().includes(searchValue.toLowerCase().trim())
```